README.md

```diff
@@ -6,14 +6,14 @@ See [License](LICENSE)
 
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](CODE_OF_CONDUCT.md)
 
-Cppfront is a experimental compiler from a potential C++ 'syntax 2' (Cpp2) to today's 'syntax 1' (Cpp1), to learn some things, prove out some concepts, and share some ideas. This compiler is a work in progress and currently hilariously incomplete... basic functions work, classes will be next, then metaclasses and lightweight exceptions.
+Cppfront is an experimental compiler from a potential C++ 'syntax 2' (Cpp2) to today's 'syntax 1' (Cpp1), to learn some things, prove out some concepts, and share some ideas. This compiler is a work in progress and currently hilariously incomplete... basic functions work, classes will be next, then metaclasses and lightweight exceptions.
 
 - [Goals and history](#goals-and-history)
 - [What's different about this experiment?](#whats-different-about-this-experiment)
 - [How do I build cppfront?](#how-do-i-build-cppfront)
 - [How do I build my `.cpp2` file?](#how-do-i-build-my-cpp2-file)
 - [Where's the documentation?](#wheres-the-documentation)
-- [Papers and talks derived from this work (presented in today's syntax)](#papers-and-talks-derived-from-this-work-presented-in-today's-syntax)
+- [Papers and talks derived from this work - presented in today's syntax](#papers-and-talks-derived-from-this-work---presented-in-todays-syntax)
```

Comment:
The issue is the apostrophe in "today's". The actual anchor doesn't have it.

Reply:
I thought I tried that, but you're totally right. Fixed in this way and squashed and force pushed